tests/MassTransit.MartenIntegration.Tests/TwoSaga_Specs.cs

```diff
@@ -49,6 +55,8 @@ public async Task Should_work_as_expected()
 
             protected void ConfigureRegistration(IBusRegistrationConfigurator configurator)
             {
+                configurator.AddLogging(loggingBuilder => loggingBuilder.AddProvider(NullLoggerProvider.Instance));
```

Comment:
The internal class MartenActivator (IHostedService) used to initialise the database has a dependency of ILogger<T> where T is the MartenActivator class, but thats internal - so had to wire up full logger.